tests/tools/archive/orm/test_codes.py

```diff
@@ -59,8 +59,6 @@ def test_input_code(aiida_profile, tmp_path, aiida_localhost):
     export_file = tmp_path / 'export.aiida'
     create_archive([calc], filename=export_file)
 
-    aiida_profile.reset_storage()
```

Comment:
This shouldn't remove here, I was wrong. It tried to reset storage to mock the behavior of import into a new storage. 
Please revert this line of change.